src/styles/index.css

```diff
@@ -0,0 +1,224 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+.body {
```

Comment:
The .body class selector is not necessary and could potentially cause confusion, as it suggests that it is styling the <body> element, which it isn't. Consider renaming this class or removing it if it's not used.